nautobot/core/fixtures/user-data.json

```diff
@@ -0,0 +1,21 @@
+[
```

Comment:
This should probably be `nautobot/core/tests/fixtures/user-data.json` if possible, to make this clear that it's only for testing.

Reply:
It's a built-in pattern for Django to look for [`fixtures` inside the app](https://docs.djangoproject.com/en/3.2/howto/initial-data/#where-django-finds-fixture-files) (similar to `templates`). I'll test to see if it can be inside the `tests` directory. If not, we can just add it to `settings.FIXTURE_DIRS`.